docs/source/private-data/private-data.md

```diff
@@ -347,6 +348,346 @@ submitted a letter of credit, prior to the execution of the transfer chaincode.
+- A hash of each price agreement is stored on the ledger. The two hashes will match only if the two organizations have agreed to the same price. This allows the organizations to verify that they have come to agreement on the transfer details before the transfer takes place. A random trade id is added to the price agreement, which serves as a *salt* to ensure that other channel members can not use the hash on the ledger to guess the price.
+
+**Step 3: Transferring the asset**
+
+After the two organizations have agreed to the same price, the asset owner can use the transfer function to transfer the asset to the buyer:
+
+- Smart contract access control ensures that the transfer must be initiated by a member of the organization that owns the asset.
+- The transfer function verifies that the asset details passed to the transfer function matches the on chain hash, to ensure that the asset owner is *selling* the same asset that they own.
+- The transfer function uses the hash of the price agreement on the ledger to ensure that both organizations have agreed to the same price.
+- If the transfer conditions are met, the transfer function adds the asset to the implicit private data collection of the buyer, and deletes the asset from the collection of the seller. The transfer also updates the owner in the public ownership record.
+- Because of the endorsement policies of the seller and buyer implicit data collections, and the state based endorsement policy of the public record (requiring the seller to endorse), the transfer needs to be endorsed by peers from both buyer and seller.
+- The state based endorsement policy of the public asset record is updated so that only a peer of the new owner of the asset can update or sell their new asset.
+
+### Running the private asset transfer smart contract
+
+You can use the Fabric test network to run the private asset transfer smart contract. The test network contains two peer organizations, Org1 and Org1, that operate one peer each. In this tutorial, we will deploy the smart contract to a channel of the test network joined by both organizations. We will first create a asset that is owned by Org1. After the two organizations agree on a asset price, we will transfer the asset from Org1 to Org2.
+
+### Deploy the test network
+
+We need to deploy an instance of the Fabric test network to run the smart contract. Open a command terminal and navigate to test network directory in your local clone of [fabric-samples](https://github.com/hyperledger/fabric-samples). We will operate from the `test-network` directory for the remainder of the tutorial.
+```
+cd fabric-samples/test-network
+```
+
+You can then deploy the network with the following command:
+```
+./network.sh up createChannel
```

Comment:
since new developers use the same network for our sample application testing as well, I think our sample applications need CA to be part of  the network (regiserUser with ca etc) and use it for cryptogen. If this is not true, I stand corrected

Making channel name explicit as well.
```suggestion
./network.sh up createChannel -ca -c mychannel
```

Reply:
Added the channel name; I did not include `-ca` as the sample is written as a standalone chaincode and does not perform any new identity registration.